core/controllers/editor_test.py

```diff
@@ -25,9 +26,12 @@
 from core.domain import stats_domain
 from core.domain import rights_manager
 from core.domain import rule_domain
+from core.platform import models
 from core.tests import test_utils
 import feconf
 
+(user_models,) = models.Registry.import_models([models.NAMES.user])
```

Comment:
@sbhowmik89 checked with me about this. In this case, the code using this is just to set up the conditions for the tests (which would be rather painful to do otherwise). There's no call to user_models directly in any of the test bodies.

So I think it's OK (but let me know if you disagree!)